modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -46,6 +46,7 @@ abstract class ParagraphsTestBase extends AbstractKernelTestBase {
     'paragraphs',
     'text',
     'typed_link',
+    'twig_field_value',
```

Comment:
remove

Reply:
removed and added on the necessary tests.